include/interface.h

```diff
@@ -111,18 +111,18 @@ class Interface : public ParsedFiniteElement<dim,spacedim>
   void apply_neumann_bcs (const typename DoFHandler<dim,spacedim>::active_cell_iterator &cell,
                           Scratch &scratch,
                           CopySystem &data,
-                          Number &energy) const;
+                          std::vector<Number> &local_residual) const;
 
 
   /**
-   * Applies Forcing terms
+   * Applies CONSERVATIVE forcing terms
```

Comment:
Actually, if you make it here, they can also be **non** conservative, since you are using the residual...